tests/Collection/CollectionFunctionalTest.php

```diff
@@ -311,6 +311,68 @@ public function testMapReduce()
         $this->assertNotEmpty($result->getCounts());
     }
 
+    /**
+     * @expectedException MongoDB\Driver\Exception\InvalidArgumentException
+     * @expectedExceptionMessage inherited
+     */
+    public function testInsertOneInTransactionWithInheritedWriteConcern()
```

Comment:
This test can change if the Collection avoids inheriting WC if it knows it would conflict with the current state of the session when `insertOne()` is called.

Reply:
Instead of changing it, I've removed it. I suppose I could replace it with a test that uses Command Observer? ``libmongoc`` doesn't seem to throw an error when we do this thing.

Reply:
I don't believe libmongoc enforces this. It may just be done server side.

An APM test to ensure we don't inherit a non-default write concern would probably be prudent.